mlflow/resource_mlflow_experiment.go

```diff
@@ -3,102 +3,64 @@ package mlflow
 import (
 	"context"
 
+	"github.com/databricks/databricks-sdk-go/service/mlflow"
 	"github.com/databricks/terraform-provider-databricks/common"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-// Experiment defines the response object from the API
-type Experiment struct {
-	Name             string `json:"name"`
-	Description      string `json:"description,omitempty"`
-	ExperimentId     string `json:"experiment_id,omitempty" tf:"computed"`
-	ArtifactLocation string `json:"artifact_location,omitempty" tf:"force_new,suppress_diff"`
```

Comment:
1. re-implement suppress-diff by implementing `DiffSuppressFunc`
2. add `ForceNew` in the `StructToSchema` callback